walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +271,36 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+    @Test
+  public void nestedEventsNonNestedFollows() {
```

Comment:
rename this to nonNestedEventFollowsNestedEvent or nestedEventsFollowedByNonNested